cfme/tests/services/test_catalog_item.py

```diff
@@ -456,3 +456,175 @@ def test_service_provisioning_email(request, appliance, catalog_item):
     provision_request.wait_for_request(method='ui')
     request.addfinalizer(provision_request.remove_request)
     assert result.validate(wait="60s")
+
+
+@pytest.mark.meta(coverage=[1746344])
+@pytest.mark.manual
+@pytest.mark.tier(2)
```

Comment:
**Suggestion:** 
I saw whole file and found that the most of tests are in `tier(2)` and very few are in `tier(1)` or `tier(3)`.
Can you please mention `tier(2)` globally and `tier(1)` and `tier(3)` on respective tests.?

Reply:
I think it will be out of the scope of this PR. will create separate pr for this. thanks for suggestions.